### PR TITLE
Update graphics-utilities.adoc

### PR DESCRIPTION
**Pull request by andrum99**
Document `pmic` subcommand to `vcgencmd measure_temp`

### CODE REVIEW

**Comment by andrum99** [comment]
Linked PR in userland repo - man page update - https://github.com/raspberrypi/userland/pull/703

File: documentation/asciidoc/computers/os/graphics-utilities.adoc [hunk 1]
```diff
@@ -138,7 +138,7 @@ Returns the throttled state of the system. This is a bit pattern - a bit being s
 
 ==== measure_temp
 
-Returns the temperature of the SoC as measured by the on-board temperature sensor.
+Returns the temperature of the SoC as measured by its internal temperature sensor. On Raspberry Pi 4 the `pmic` subcommand can be used to instead return the temperature of the PMIC.
```
Comment (lurch) [inline documentation/asciidoc/computers/os/graphics-utilities.adoc:141]: It's not entirely clear if this means `vcgencmd pmic` or `vcgencmd measure_temp pmic`?
Also, I may just be reading it wrong but the "instead" implies that the former option isn't available and only the latter option is available? Maybe it's worth rewording to use "also" to clarify that both options are available, IYSWIM?

Comment (andrum99) [reply documentation/asciidoc/computers/os/graphics-utilities.adoc]: I've changed the wording slightly. 

(A quick test on a Pi 3B+ and Pi 1 reveals that on pre-Pi4 boards, `vcgencmd measure_temp pmic` will silently ignore the `pmic` parameter and just return the temperature of the SoC).

**Comment by aallan** [comment]
Ready to merge?

**Review by lurch — APPROVED** [review]
LGTM

**Pull request merged**